README.md

```diff
@@ -770,7 +770,7 @@ Work in progress drivers. Help the authors make these crates awesome!
 [DS3231]: https://github.com/wose/ds3231
 [DS3234]: https://github.com/rust-embedded/wg/issues/39#issuecomment-375262785
 [DS323x]: https://crates.io/crates/ds323x
-[eink-waveshare]: https://crates.io/crates/eink_waveshare_rs
+[epd-waveshare]: https://crates.io/crates/epd-waveshare
```

Comment:
You will also need to update the link referring to this label, right?

Reply:
Sorry you're right !